site/public/css/server.css

```diff
@@ -628,6 +628,24 @@ td > a.active-sort {
     padding: 5px 5px 10px;
 }
 
+.no-border-box .col-md-4 {
+    float: right;
+    word-break: break-word;
+    display: inline-block;
+}
+
+.no-border-box .col-md-4 .offset-md-4 {
+    float: center;
+    word-break: break-word;
+    display: inline-block;
+}
```

Comment:
The `float` property is conflicting in both of these. VSCode also warns me that `display: inline-block` is ignored because of specifying a value for `float`. I think it would be better to separate these 3 classes differently and avoid specifying something like 2 different values for `float` in 2 different places for the same class.

Reply:
The saving issue is due to the permission of the annotation folder. I just fixed it in #8176 . Thanks for letting me know.